pages/1.11/release-notes/1.11.0-rc4/index.md

```diff
@@ -68,7 +68,7 @@ Please try out the new features and updated data services. Provide any feedback
 - Unified Logging Endpoint to Collect Container (task) as well as System Component Logs.
 
 ## Storage
-- DC/OS 1.11 introduces an implementation of the industry-standard Container Storage Interface (CSI) version 0.1, which enables developers (Mesosphere, community, and partners) to streamline the development of storage features within DC/OS by providing a common API between the Container Orchestrator (DC/OS) and the storage devices. [View the documentation](/services/beta-storage/0.1.0-beta). [enterprise type="inline" size="small" /]
+- DC/OS 1.11 introduces an implementation of the industry-standard Container Storage Interface (CSI) version 0.1, which enables developers (Mesosphere, community, and partners) to streamline the development of storage features within DC/OS by providing a common API between the Container Orchestrator (DC/OS) and the storage devices. [View the documentation](/services/storage/latest). [enterprise type="inline" size="small" /]
```

Comment:
since beta-storage is no longer supported, and the docs removed, I wonder if this release note needs to be re-written? same goes for all other, similar 1.11.x and 1.12.x release notes that reference beta-storage (and only work with beta-storage)

Reply:
I think that's ok to have the 1.0 release notes updated to state that beta versions of DSS or those versions below 1.0 are not supported.

Reply:
ok, that sounds like a doc change that we'd need to make upstream and then pick into the 1.0 docs here